rbac/management/seeds.py

```diff
@@ -19,14 +19,19 @@
 import logging
 from functools import partial
 
-from django.db import connection, connections
+from django.db import connections
 from management.cache import AccessCache
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
-def on_complete(completed_log_message, future):
+def on_complete(completed_log_message, tenant, future):
     """Explicitly close the connection for the thread."""
+    logger.info("Purging policy cache.")
```

Comment:
It looks to me, the code was written before, it did not purge the cache for all tenants. I set this to have a callback that once the new data was in the database and committed to purge the tenant's cache. Perhaps I'm misreading?

Reply:
Hm, the way it was, it would just purge the cache once for all tenants after each seeding (all roles/all groups) ran. We did confirm that was happening, but maybe there was something else amiss? 

If this is the direction we're moving, we could probably set the signature back to pass in the actual tenant?

Reply:
I'm looking at the `finally:` block that used to be at line 71 below - how would that remove the cache for all tenants versus whatever the currently active schema is?

Reply:
They `keys_for` value in that case ends up being:
```
rbac::policy::tenant=*::user=*
```